api/Makefile

```diff
@@ -384,6 +384,9 @@ setup-soap-proxy:
 validate-simpler-endpoints:
 	$(PY_RUN_CMD) validate-simpler-endpoints
 
+setup-cert-user:
+	$(PY_RUN_CMD) flask task setup-cert-user
```

Comment:
How do you pass in the parameters to this command?

I'd actually say we should not add this to avoid bloating the makefile / the primary way this will be run is against an environment non-locally which is an entirely different command, and those aren't in the makefile either.

For running locally, `make cmd args="task setup-cert-user --tcertificates-id X --role-ids X"` would do this.